spec/attachment2xml.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-env node, jasmine */
+'use strict';
+
+const attachmentToJson = require('../lib/actions/attachmentToJson');
+const json = require('./data/po.json');
```

Comment:
Does `require` work for a JSON file?

Reply:
it was used like this in the other test that were already made